configs/yolo/yolov3_ms_aug_273e.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
+
+# model settings
+model = dict(
+    type='YOLOV3',
+    pretrained='./work_dirs/darknet_state_dict_only.pth',
+    backbone=dict(
+        type='Darknet',
+        depth=53,
+        out_indices=(3, 4, 5),
+    ),
+    neck=dict(
+        type='YOLOV3Neck',
+        num_scales=3,
+        in_channels=[1024, 512, 256],
+        out_channels=[512, 256, 128],
+    ),
+    bbox_head=dict(
+        type='YOLOV3Head',
+        num_classes=80,
+        num_scales=3,
+        num_anchors_per_scale=3,
+        in_channels=[512, 256, 128],
+        out_channels=[1024, 512, 256],
+        strides=[32, 16, 8],
+        anchor_base_sizes=[
+            [(116, 90), (156, 198), (373, 326)],
+            [(30, 61), (62, 45), (59, 119)],
+            [(10, 13), (16, 30), (33, 23)],
+        ],
+    ))
+# training and testing settings
+train_cfg = dict(
+    one_hot_smoother=0., ignore_config=0.5, xy_use_logit=False, debug=False)
+test_cfg = dict(
+    nms_pre=1000,
+    min_bbox_size=0,
+    score_thr=0.05,
+    conf_thr=0.005,
+    nms=dict(type='nms', iou_thr=0.45),
+    max_per_img=100)
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255., 255., 255.], to_rgb=True)
+# TODO: Add PhotoMetricDistortion
```

Comment:
We may remove this #TODO.

Reply:
Sure, I will add PhotoMetricDistortion, just like what darknet does.

Reply:
Sure.